app/preview/TemplateParams.scala

```diff
@@ -1986,104 +1986,12 @@ object TemplateParams2 {
       "verificationLink" -> exampleLinkWithRandomId,
       "name"             -> "Susan Bartlett"
     ),
-    "tdq_compliance_all_required_headers_missing" -> Map(
```

Comment:
These were for our old templates which I deleted in a previous PR but forgot to remove these at the time.